charts/tidb-cluster/templates/monitor-deployment.yaml

```diff
@@ -207,11 +207,11 @@ spec:
           - key: dashboard-config
             path: dashboards.yaml
         name: dashboards-provisioning
+      {{- end }}
```

Comment:
I think it should include `prometheus-rules` when not creating Grafana.

Reply:
it does. The helm chart is broken now: I would suggest merging this and creating a follow up issue for the changes you would like to make.

Reply:
This should solve the problem: 
```yaml
      - emptyDir: {}
        name: grafana-dashboard
      {{- end }}
      - emptyDir: {}
        name: prometheus-rules
```

Reply:
That would break the initializer job.

Honestly I am afraid to make any extra changes to our helm charts now that there are so many options that are unverified.